improver_tests/cli/test_init.py

```diff
@@ -192,10 +196,49 @@ def test_with_output(self, m):
         m.assert_called_with(True, 'foo')
         self.assertEqual(result, 4)
 
+# pylint: disable=W0613
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'W0613' is used to disable 'unused-argument' message emission](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5066912)